vsintegration/tests/UnitTests/InlineParameterNameHintTests.fs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+namespace VisualFSharp.UnitTests.Editor.Hints
+
+open NUnit.Framework
+open HintTestFramework
+
+module InlineParameterNameHintTests =
```

Comment:
Please add a test showing how this works for generics.
E.g. an inline function doing addition of two numbers, can be both ints as well as floats.